timesketch/tsctl.py

```diff
@@ -439,6 +455,6 @@ def export_sigma_rules(path):
             continue
 
         with open(file_path, "wb") as fw:
-            fw.write(rule.rule_yaml.encode("utf-8"))
+            fw.write(rule.rule_yaml.encode('utf-8'))
```

Comment:
Please setup Black for formatting. Singel quotes are not consistent with the rest of the codebase.